api/database/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM postgres:alpine
+COPY *.sql /docker-entrypoint-initdb.d/
+ADD scripts/1_init.sql /docker-entrypoint-initdb.d
```

Comment:
These are the wrong paths for these scripts. It looks like all of the database import stuff was committed mid-move from a different folder.

Reply:
fixed